ddp-workspace/projects/ddp-dsm-ui/src/app/dashboard-statistics/components/date-range/date-range.component.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  OnDestroy,
+  OnInit,
+  Output
+} from '@angular/core';
+import {AbstractControl, FormControl, FormGroup, ValidationErrors, Validators} from '@angular/forms';
+import {DateRangeModel} from '../../models/DateRange.model';
+import {Subject} from 'rxjs';
+import {takeUntil, auditTime} from 'rxjs/operators';
+import {DatePipe} from '@angular/common';
+import {Global} from '../../../globals/globals';
+
+@Component({
+  selector: 'app-date-range',
+  templateUrl: 'date-range.component.html',
+  styleUrls: ['date-range.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DateRangeComponent implements OnInit, OnDestroy {
+  /**
+   * @MAIN Main Date Range Object
+   */
+  public readonly dateRangeForm: FormGroup = this.generateFormGroup;
```

Comment:
**#optional**

I'd rather have initiation here, like
`public readonly dateRangeForm: FormGroup = new FormGroup(...)`

It seems a bit more readable (for me).

Reply:
Agree. moved